### PR TITLE
Added working directory command in dockerfile

### PR DESCRIPTION
**Pull request by Falguni-M**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
Inserted a line in Dockerfile, confirming the working directory.

### CODE REVIEW

File: kubernetes/Dockerfile [hunk 1]
```diff
@@ -14,6 +14,7 @@ RUN mkdir /data && mkdir /app && mkdir /home/checkov
 RUN chown checkov:checkov /data /app /home/checkov
 
 COPY kubernetes/run_checkov.sh /app
+WORKDIR /app
```
Comment (schosterbarak) [inline kubernetes/Dockerfile:17]: why is this important?

**Comment by Falguni-M** [comment]
The WORKDIR instruction sets the working directory for any command(RUN, CMD, etc) that follows it in the Dockerfile. If the WORKDIR doesn’t exist, it will be created even if it’s not used in any subsequent Dockerfile instruction.
Docker best practices recommend you to use it. It's totally fine to not use it as well since the "RUN chown checkov:checkov /data /app /home/checkov" command will do it automatically.

**Comment by schosterbarak** [comment]
got ya. just read about it. thanks @Falguni-M

**Comment by Falguni-M** [comment]
Thanks for letting me contribute! :)

**Pull request merged**